WPMedia/fusion-news-theme-blocks
# Primary font refactor

**If you have not filled out the checklist below, the pr is not ready for review.**

## Description
This is an overall refactor to our use of font in the blocks library. Essentially, this should replace where possible all uses of of the `Styled Components` library and replace with the `PrimaryFont` tag from our `shared-styles` lib.

## Jira Ticket
- [TMEDIA-202](https://arcpublishing.atlassian.net/browse/TMEDIA-202)

## Acceptance Criteria
Refactor the use of styled components and where possible, take that dependency from individual block `package.json` files.
Tests should all pass.

## Test Steps
- Add test steps a reviewer must complete to test this PR

1. Checkout this branch `git checkout TMEDIA-202/primary_font_refactor`
2. Run fusion repo with any linked blocks: `npx fusion start -f -l @wpmedia/masthead-block`
3. Go to http://localhost/pagebuilder/editor/curate?p=pnLrGpJS6maOB88qs&v=vhtWpFUTNxaOB88qs
3. Compare blocks changed here to CoreComponents blocks page: https://corecomponents.arcpublishing.com/pagebuilder/editor/curate?p=pnLrGpJS6maOB88qs&v=vhtWpFUTNxaOB88qs
4. Check that overall font usage is correct throughout blocks, only the blocks changed need to be gone over:

Blocks to check:

- Card list block
- Date block
- Footer block
- Header block
- Masthead block
- Tag title block
- Video player block
- Video promo block
- Author Bio block
- Full author bio block
- Links bar block
- Subheadline block

## Effect Of Changes
No effects

## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist
_The reviewer of the PR should copy-paste this template into the review comments on review._
